### PR TITLE
LB-122: Fix the issue of album_msid and album_name getting dropped during influx write

### PR DESCRIPTION
**Pull request by paramsingh**
The code was looking for `album_msid` instead of `release_msid`
leading to nothing being stored in the album_msid field in influx.

### CODE REVIEW

**Comment by paramsingh** [comment]
Note: This partly fixes LB-122 as album names are still not handled. However I think some discussion is required on where release_name belongs exactly, as the changes will have to be made according to that.

File: listen.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def from_json(cls, j):
             user_name=j.get('user_name', ""),
             timestamp=datetime.utcfromtimestamp(float(j['listened_at'])),
             artist_msid=j['track_metadata']['additional_info'].get('artist_msid'),
-            album_msid=j['track_metadata']['additional_info'].get('album_msid'),
```
Comment (alastair) [inline listen.py]: How complex is to to replace `album_*` with `release_*` everywhere in the `Listen` class and places that use it?

Comment (paramsingh) [inline listen.py]: From a look at the codebase, seems like `album_*` is used in the listenstores (both postgres and influx) and the bigquery schema has columns named `album_*`. Wouldn't be too complex to change it all to `release_*`, although I was thinking maybe there is some reason we moved from `release_*` to `album_*` related to MB? If there isn't, a refactor wouldn't be much effort I think.

Comment (alastair) [inline listen.py]: I suspect it's because listenstore was originally written with `album`. The rest of listenbrainz used `release`, but we've probably been slowly building out features using listenstore as a base.
I'd confirm with @mayhem, but my opinion is that we should use the MusicBrainz standard here, which is release, not album.

Comment (paramsingh) [inline listen.py]: Waiting for @mayhem for the final say. I think this refactor should be a different PR anyways.

**Comment by paramsingh** [comment]
I have fixed the dropping of `album_name` also now, after some discussion with @alastair where we both agreed that `release_name` is better placed in the top level and the docs also mention multiple times that `release_name` should be in the top level.

**Comment by alastair** [comment]
Create a ticket to rename album->release in the code, and then I'll approve this 💿

**Comment by paramsingh** [comment]
@alastair, I have created [LB-125](https://tickets.metabrainz.org/browse/LB-125).

The work involved isn't much and should be easy to do. I've started with it and should have the changes done and tested by today hopefully. Mostly will involve changes in the ListenStore and the Listen class.

**Comment by mayhem** [comment]
Bug created, looks fine, merging.
